fili-generic-example/src/test/java/com/yahoo/wiki/webservice/data/config/auto/StaticWikiConfigLoader.java

```diff
@@ -44,7 +44,7 @@ public TableName getTableName() {
             }
 
             @Override
-            public List<DataSegment> getDataSegmentMetadata() {
+            public List<DataSegment> getDataSegments() {
```

Comment:
This is not a new method. It's the old method that's been renamed. The reason is mentioned in previous comments.